src/main/java/io/appform/dropwizard/actors/base/Handler.java

```diff
@@ -58,12 +70,19 @@ public void handleDelivery(String consumerTag, Envelope envelope,
                                AMQP.BasicProperties properties, byte[] body) throws IOException {
         try {
             final Message message = mapper.readValue(body, clazz);
-            boolean success = retryStrategy.execute(() -> handle(message, messageProperties(envelope)));
+            val tracer = GlobalTracer.get();
+            Span childSpan = buildChildSpan(properties, tracer);
+            log.info("publishing message with traceId: {}, spanId: {}", childSpan.context().toTraceId(),childSpan.context().toSpanId());
+            try (Scope scope = tracer.scopeManager().activate(childSpan)) {
```

Comment:
what if tracer is null?

Reply:
we should also catch any exceptions related to tracing so that any issue in tracing doesn't impact the normal functionality

Reply:
all logic is inside try-catch now